src/test/regress/expected/shard_rebalancer.out

```diff
@@ -1286,6 +1298,10 @@ SELECT * FROM public.table_placements_per_node;
 SELECT * FROM rebalance_table_shards('tab', rebalance_strategy := 'by_disk_size', shard_transfer_mode:='block_writes', threshold := 0);
 WARNING:  the given threshold is lower than the minimum threshold allowed by the rebalance strategy, using the minimum allowed threshold instead
 DETAIL:  Using threshold of 0.01
+NOTICE:  skipping child tables for optimization
```

Comment:
Here we give identical logs and having one of this vs 1000s of this log probably doesn't make much difference for the users, as in what will they understand? Would that make sense to add the relation name that corresponds to `shardInterval->relationId` or something?

Reply:
Makes sense